README.md

```diff
@@ -142,6 +142,10 @@ wrapper of `ikfast.cpp` to the kinematics base interface of moveit.
 Contains a specialized version of `ros_controllers::JointTrajectoryController` which can be put into a holding mode.
 A controlled stop using a hold trajectory is performed thus stopping the manipulator without the mechanical stress of a hard brake.
 
+**Please note:**
+Due to safety reasons, the command interface of the `PilzJointTrajectoryController` is deactivated. For more information
+see issue [#493](https://github.com/ros-controls/ros_controllers/issues/493) on ros-controls/ros_controllers.
```

Comment:
The reason is "single-point-of-control" and the ability to abort a goal (using the action interface).

Reply:
Imho "single-point-of-control" is a much broader concept. Which needs to be explained in a more general fashion.

Reply:
> The reason is "single-point-of-control" and the ability to abort a goal (using the action interface).

If you have a concrete message in mind, please state the message.

Reply:
In order to keep focus and move on I suggest deferring the "single-point-of-control" 
conceptualization and required holistic discussion to a later point in time. K.I.S.S.!

Reply:
Resolving for focus.